Refine serialization of info fields for Equity

rsmb7z opened the pull request:
# Pull Request

Refine serialization of info fields for Equity

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

cjdsellers commented:
Thanks for this and it's a good standardization.

I think a test `catalog` is accidentally being committed under `unit_tests` though? 
I've updated the `.gitignore` to avoid this.

filipmacek commented:
@cjdsellers looks good to me, I missed this field in last weeks PR https://github.com/nautechsystems/nautilus_trader/pull/1542

rsmb7z commented:
> I think a test `catalog` is accidentally being committed under `unit_tests` though? I've updated the `.gitignore` to avoid this.

Yes by mistake and removed it now.
